lib/RESTClient.js

```diff
@@ -393,6 +393,37 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   Get raft logs from bucketd
+    *   get server's response and return it
+    *
+    *   @param {string} raftId - raft session id
+    *   @param {number} [start=1] - starting sequence number
+    *   @param {number} [end=10000] - ending sequence number
+    *   @param {boolean} [targetLeader=false] - true: from leader instead of
+    *       follower
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getRaftLog(raftId, start, end, targetLeader, reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = `/_/raft/${raftId}/log`;
```

Comment:
should it not be `/_/raft_sessions/${raftId}/log` instead ? This path looks like a repd admin path rather than a Bucketd one to me.